ethicalads-theme/templates/ea/homepage.html

```diff
@@ -87,10 +86,10 @@ <h3>Focused and relevant</h3>
         </div>
 
         <!-- Heading -->
-        <h3>Respectful</h3>
+        <h3>Respectful &amp; private</h3>
 
         <!-- Text -->
-        <p class="text-gray-800 mb-0">We believe in a better internet. That's why we support AcceptableAds, BetterAds, and DoNotTrack. We provide transparent reporting for tech marketers, while respecting user privacy.</p>
+        <p class="text-gray-800 mb-0">We show high-performing ads to developers with <strong>no invasive tracking</strong> and no cookie banners while providing transparent reporting for tech marketers.</p>
```

Comment:
This is a long sentence, and feels like it's trying to do too much. `no invasive tracking` also feels like not the benefit we want to sell. Publishers get cookie banners, not sure the advertiser benefit of privacy here? Not 100% in love with my copy, but I don't think the original is right. 

```suggestion
        <p class="text-gray-800 mb-0">Your site deserves beautiful, developers-focused ads with <strong>no cookie banners</strong>. Everyone gets transparent reporting on exactly what is shown where.</p>
```